src/Host.cpp

```diff
@@ -46,7 +46,9 @@
 #include <memory>
 #include "post_guard.h"
 
-const int Host::mLuaSendPasswordTimeout = 30000; // milli-seconds!
+using namespace std::chrono_literals;
+
+const int Host::mLuaSendPasswordTimeout = 30s;
```

Comment:
Sorry, it shouldn't be an int, try auto.

Reply:
And what goes in for the declaration in the header file?